src/services/organisation/index.js

```diff
@@ -0,0 +1,145 @@
+const joi = require('@hapi/joi')
+const slugify = require('slugify')
+const uuid = require('uuid/v4')
+const { applyFiltersToQuery } = require('../../utils/databaseUtils')
+const { NotFoundError } = require('../../utils/errors')
+const Boom = require('@hapi/boom')
+const User = require('../../models/v2/User')
+const Organisation = require('../../models/v2/Organisation')
+
+const getOrganisationQuery = db =>
+    db('organisation')
+        .select(
+            'organisation.id',
+            'organisation.uuid',
+            'organisation.name',
+            'organisation.slug',
+            db.ref('u.uuid').as('created_by_user_uuid'),
+            db.ref('u.id').as('created_by_user_id')
+        )
+        .innerJoin(
+            db.ref('users').as('u'),
+            'organisation.created_by_user_id',
+            'u.id'
+        )
+
+async function create({ userId, name }, db) {
```

Comment:
why the mix of syntax in function declarations between `const xxx => ... ` and  `async function xxx()` ? I suggest we keep it to the same format throughout the code for all functions no matter if async or not. In most places in the backend here we're using the first variant of those

Reply:
Not really a reason, I think it was just because I extracted it from a function. You would prefer to refactor the other functions to use const instead?

Reply:
yeah I think so, it would at least be more consistent with other parts of the project (and I personally think it's more good looking :) )